TOOODO/Data/Sources/Dto/Local/CategoryDto.swift

```diff
@@ -7,29 +7,40 @@
 
 import RealmSwift
 import Domain
+import Foundation
 
 public class CategoryDto: Object {
     @Persisted(primaryKey: true) var id: ObjectId
     @Persisted var category: String
     @Persisted var categoryDes: String
     @Persisted var color: Int
     @Persisted var todos: List<TodoDto>
+    @Persisted var date: Date
 }
 
 extension CategoryDto {
-    func toDomain() -> Category {
+    func toDomain() -> Domain.Category {
         Category(id: self.id.stringValue, category: self.category, categoryDes: self.categoryDes, color: self.color, todos: self.todos.map{ $0.toDomain() })
```

Comment:
```suggestion
        Category(
           id: self.id.stringValue, 
           category: self.category, 
           categoryDes: self.categoryDes, 
           color: self.color, 
           todos: self.todos.map{ $0.toDomain() }
           )
```
한 줄 보다는 이렇게 하는게 가독성이 더 좋아요